README.md

```diff
@@ -943,6 +943,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Siddharth Singh Tanwar](https://github.com/SiddharthSinghTanwar)
   - [Silas Omurunga](https://github.com/Simbadeveloper)
   - [Simran Biswas](https://github.com/simranbiswas)
+  - [Sidra Waseem](https://github.com/sidramwaseem)
```

Comment:
Please add your name in line 944, after Siddharth Singh Tanwar.
Let us know if you need any help doing that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable